Makefile

```diff
@@ -63,4 +63,11 @@ debug-python: activate-storage
 	cd python && maturin develop --features=storage --extras=dev
 
 python-docs:
-	cd docs && make html
\ No newline at end of file
+	cd docs && make html
+
+docker-base-build:
+	cd docker/base && docker build --platform linux/amd64 -t pometry/raphtory_base .
```

Comment:
the `--platform` arg has the potential of wasting the time of devs hoping to get a local docker instance,
maybe specify the platform in the name of the make task? `docker-base-build-x86`